app/jobs/cancel_outdated_reservations_job.rb

```diff
@@ -0,0 +1,27 @@
+class CancelOutdatedReservationsJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    reservations.each do |reservation|
+      cancel_reservation(reservation, status).call unless outdated?(reservation)
```

Comment:
you don't need to pass `status` here :D it can be called in `cancel_reservation` method :)

```ruby
  def cancel_reservation(reservation)
    Reservations::UseCases::Update.new(reservation: reservation, status: status)
  end
  ```

Reply:
it even is called in cancel_reservation🙈